src/blog/2018-12-24/stryker4s-0-1-0-released.pug

```diff
@@ -0,0 +1,30 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+        Stryker4s just had it's first official release! In this blog post we'll tell you about why running with the sbt plugin is different, and some of the features you can use today for Stryker4s.
```

Comment:
I assume you can still use Scala4s in non-sbt-projects? Maybe it's a good idea to reiterate that.

> Although it's possible to use Scala4s in non-sbt projects (link to quickstart?), we decided to introduce it as an SBT plugin for the first release.